docs/components/movement-sensor/_index.md

```diff
@@ -40,57 +40,13 @@ Model | Supported hardware <a name="model-table"></a>
 
 ## Control your movement sensor with Viam's client SDK libraries
 
-The following example assumes you have a movement sensor called `my-imu` configured as a component of your robot.
-If your movement sensor has a different name, change the `name` in the example.
+To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com), navigate to the **code sample** tab, select your preferred programming language, and copy the sample code generated.
 
-{{< tabs >}}
-{{% tab name="Python" %}}
-
-Place the example code in `main()` after `robot = await connect()`.
-
-```python
-from viam.components.movement_sensor import MovementSensor
-
-robot = await connect() # Refer to CODE SAMPLE tab code
-my_imu = MovementSensor.from_robot(robot, "my-imu")
-
-# Get the current linear acceleration
-lin_accel = await my_imu.get_linear_acceleration()
-print(f"my-imu linear acceleration in the x direction: {lin_accel.x}")
-```
-
-{{% /tab %}}
-{{% tab name="Go" %}}
-
-Place the example code in the `main()` function after `robot, err := client.New(...)`.
+When executed, this sample code will create a connection to your robot as a client.
+Then control your robot programmatically by adding API method calls as shown in the following examples.
 
-```go
-import (
-  "context"
-  "time"
-
-  "github.com/edaniels/golog"
-
-  "go.viam.com/rdk/components/movementsensor"
-)
-
-robot, err := client.New() // Refer to CODE SAMPLE tab code
-// Grab the movement sensor from the robot
-myIMU, err := movementsensor.FromRobot(robot, "my-imu")
-if err != nil {
-  logger.Fatalf("cannot get movement sensor: %v", err) 
-}
-
-// Get the current linear acceleration
-linAccel, err := myIMU.LinearAcceleration(context.Background(), map[string]interface{}{})
-if err != nil { 
-  logger.Fatalf("cannot get linear acceleration: %v", err) 
-}
-logger.Info("my-imu LinearAcceleration in the x direction: ", linAccel.X)
-```
-
-{{% /tab %}}
-{{< /tabs >}}
+These examples assume you have a movement sensor called `"my_movement_sensor"` configured as a component of your robot.

```

Comment:
Good call changing it from my_imu but it could stay my_sensor for brevity if we're not rigidily tied to the full component name (as it looks like we're not in input controller?)

Reply:
Do you think it could be confusing with the sensor code samples / if they had a sensor configured separately and used code sample tab?